### PR TITLE
Set up .env file, declare User types, create User/Elderly/Caretaker Context

### PR DESCRIPTION
**Pull request by chuwong35122**
**Note**
- .env is according to this [package](https://www.npmjs.com/package/babel-plugin-inline-dotenv)

### CODE REVIEW

File: src/dto/modules/modules.dto.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  gender: string;
+  bday: string;
+  bloodType: BloodType;
+  healthCondition: string | null;
+  personalMedication: string | null;
+  allergy: string | null;
+  vaccine: string | null;
+  phone: string;
+  listModuleid: ModuleId[];
+}
```
Comment (pointmekin) [inline src/dto/modules/modules.dto.ts]: I think we can create `CaretakerUser`  then extend it to create `ElderlyUser`
```ts
export interface CaretakerUser {
  fname: string;
  lname: string;
  dname: string;
  gender: string;
  bday: string;
  phone: string;
}

export interface ElderlyUser extends CaretakerUser {
  bloodType: BloodType;
  healthCondition: string | null;
  personalMedication: string | null;
  allergy: string | null;
  vaccine: string | null;
  listModuleid: ModuleId[];
}
```

**Comment by pointmekin** [comment]
Please update the interfaces to these as both elderly and caretaker users can have the optional health information

```ts
import { ModuleId } from './modules.dto';

export type BloodType = 'N/A' | 'A' | 'B' | 'O' | 'AB' | string;

export enum Gender {
  female = 'F',
  male = 'M'
}
export interface User {
  imageid: string;
  fname: string;
  lname: string;
  dname: string;
  bday: string;
  gender: 'F' | 'M';
  isElderly: boolean;
  uid: number;
  healthCondition?: string | null;
  bloodType: BloodType;
  personalMedication?: string | null;
  allergy?: string | null;
  vaccine?: string | null;
  phone?: string;
}

export interface Caretaker extends User {
  listElderly: Elderly[];
}

export interface Elderly extends User {
  listModuleid: ModuleId[];
}
```

File: src/dto/modules/user.dto.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  imageid: string;
+  fname: string;
+  lname: string;
+  dname: string;
+  bday: string;
+  gender: 'F' | 'M';
+  isElderly: boolean;
+  uid: number;
+  healthCondition?: string | null;
+  bloodType: BloodType;
```
Comment (pointmekin) [inline src/dto/modules/user.dto.ts:19]: I think bloodType could be returned as a null value if the user doesn't include it initially. 
Consider changing to `bloodType?: BloodType | null` ?

**Review by pointmekin — APPROVED** [review]

File: src/dto/modules/user.dto.ts (hunk 2 above)
Comment (chuwong35122) [reply src/dto/modules/user.dto.ts:19]: The default value should be 'N/A', so no need :D

**Pull request merged**